x-pack/plugins/apm/public/components/app/ErrorGroupDetails/Distribution/index.tsx

```diff
@@ -4,31 +4,24 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import {
+  Axis,
+  Chart,
+  HistogramBarSeries,
+  niceTimeFormatter,
+  Position,
+  ScaleType,
+  Settings,
+  SettingsSpec,
+  TooltipValue,
+} from '@elastic/charts';
 import { EuiTitle } from '@elastic/eui';
-import theme from '@elastic/eui/dist/eui_theme_light.json';
-import numeral from '@elastic/numeral';
-import { i18n } from '@kbn/i18n';
 import d3 from 'd3';
-import { scaleUtc } from 'd3-scale';
-import { mean } from 'lodash';
 import React from 'react';
 import { asRelativeDateTimeRange } from '../../../../../common/utils/formatters';
-import { getTimezoneOffsetInMs } from '../../../shared/charts/CustomPlot/getTimezoneOffsetInMs';
-// @ts-expect-error
-import Histogram from '../../../shared/charts/Histogram';
-import { EmptyMessage } from '../../../shared/EmptyMessage';
-
-interface IBucket {
-  key: number;
-  count: number | undefined;
-}
-
-// TODO: cleanup duplication of this in distribution/get_distribution.ts (ErrorDistributionAPIResponse) and transactions/distribution/index.ts (TransactionDistributionAPIResponse)
-interface IDistribution {
-  noHits: boolean;
-  buckets: IBucket[];
-  bucketSize: number;
-}
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
Can this be deleted?

Reply:
No, even though it is a typed import eslint still complains about importing something from `server`.

Reply:
ah, ok 👍